packages/passport/sdk-sample-app/src/components/Message.tsx

```diff
@@ -17,7 +17,7 @@ function Message() {
   return (
     <CardStack title="Message">
       <Sticker
```

Comment:
@glomotion should this version upgrade have any impact on the Sticker component?
Left is after this version bump, right is prior

![image](https://github.com/user-attachments/assets/7bded0cd-ae25-4123-9c3c-b45b997c1d22)

Reply:
@haydenfowler so stickers have not changed, but Sticker.FramedIcon has. 😅 Think we just need one extra prop here. Ill fix.

Reply:
OK this commit should fix it: https://github.com/immutable/ts-immutable-sdk/pull/2219/commits/819e41e162858f81a93b6d3f55860ba1e4b9bdda

Reply:
@glomotion Same result unfortunately 😅

Reply:
@haydenfowler OK should definitely be fixed this time. Sorry for the false start. 🤪